specification/app/resource-manager/Microsoft.App/stable/2025-01-01/CommonDefinitions.json

```diff
@@ -491,6 +491,18 @@
           "type": "integer",
           "default": 10
         },
+        "cooldownPeriod": {
+          "format": "int32",
+          "description": "Optional. KEDA Cooldown Period. Defaults to 300 seconds if not set.",
+          "type": "integer",
+          "default": 300
+        },
+        "pollingInterval": {
```

Comment:
>pollingInterval [](http://example.com/codeflow?start=9&length=15)

mention units in the name

Reply:
updating name will be breaking change compared with previous api version. I updated description for unit info.

Reply:
Isnt this a new property being introduced? Why would this be a breaking change?